src/libs/dutil/WixToolset.DUtil/butil.cpp

```diff
@@ -147,7 +123,7 @@ DAPI_(HRESULT) BundleEnumRelatedBundle(
     HKEY hkUninstall = NULL;
     HKEY hkBundle = NULL;
     LPWSTR sczUninstallSubKey = NULL;
-    DWORD cchUninstallSubKey = 0;
+    size_t cchUninstallSubKey = 0;
```

Comment:
We usually use `SIZE_T`, for whatever reason...

Reply:
If it's a public parameter I used `SIZE_T` to follow our other APIs. Otherwise, I used `size_t` because the Windows APIs take `size_t` which is not implicitly convertible from `SIZE_T`.